pyanaconda/modules/storage/bootloader/extlinux.py

```diff
@@ -61,12 +62,12 @@ def write_config_console(self, config):
         console_arg = "console=%s" % self.console
         if self.console_options:
             console_arg += ",%s" % self.console_options
-        self.boot_args.add(console_arg)
+        self.boot_args.update([console_arg])
 
     def write_config_images(self, config):
         self.write_config_console(config)
         for image in self.images:
-            args = Arguments()
+            args = BootLoaderArguments()
```

Comment:
Could you please write some unit tests and test them on the `Arguments` class first to make sure that the `BootLoaderArguments` class works the same way?

Reply:
I guess this was the smart thing to start with :-/

Reply:
Done.